tooling/template-base/components/Signin/SigninPage.tsx

```diff
@@ -1,12 +1,29 @@
-import React, { useState } from 'react';
+import React, { useState, FormEvent } from 'react';
+import { gql, useMutation } from '@apollo/client';
 import Logo from '../Logo/Logo';
 
+const LoginDocument = gql`
```

Comment:
use inline graphql document here. 

fyr, also tried using the `useLoginMutation` from `@thoughtindustries/user`, ran into some strange server-bundle issue. The error `ReferenceError: window is undefined` only occurs to the deployed helium app and renders the `signin` page with 500 error. It seems that the module `html-dom-parser` could be the cause, which offers bundles for both browser and server environments. When built with wrangler v1 and webpack, the worker bundle is built with the browser version of the module (default behavior when webpack targets `webworker`, that it refers to the `browser` field of the module.) couldn't figure out a solution to this yet.

Reply:
Note: discussed offline; going to handle update of `@thoughtindustries/user`'s `useLoginMutation` in a separate ticket, as there is a likely possibility of larger changes that would be out of scope for this ticket.